lib/resize.js

```diff
@@ -1,19 +1,23 @@
-const Jimp = require('jimp');
-const TinyColor = require('tinycolor2');
+const sharp = require('sharp');
 
 module.exports = async ({ background, type = 'resize', width, height }, buffer) => {
+  type = type === 'resize' ? 'cover' : type;
   // if only one of width/height was provided, make it a square or let Jimp decide if it is able
-  if (width && !height) {
-    height = type === 'resize' ? Jimp.AUTO : width;
+  const resizeOptions = { fit: type };
+  if (width) {
+    resizeOptions.width = width;
   }
-  if (height && !width) {
-    width = type === 'resize' ? Jimp.AUTO : height;
+  if (height) {
```

Comment:
similar to the comment above ... what happens in sharp if only one dimension is passed? Need to make sure it's handled.

Reply:
Sharp automatically resolves the missing dimension.